.travis.yml

```diff
@@ -4,8 +4,8 @@ go:
   - 1.14.x
 
 install:
-  - curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | BINARY=golangci-lint sh -s -- -b $(go env GOPATH)/bin v1.21.0
-
+  - curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | BINARY=golangci-lint sh -s -- -b $(go env GOPATH)/bin v1.27.0
+  - curl -sfL https://raw.githubusercontent.com/securego/gosec/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v2.3.0
 script:
```

Comment:
We are installing gosec, but we also need to run it on scripts part

Reply:
> We are installing gosec, but we also need to run it on scripts part

So I need to add the same gosec rule to makefile? To make it work here

Reply:
We need to run gosec here on `scripts` as we are running linters+tests